### ISSUE TITLE
Use a different colour for 00, FF chains

### ISSUE THREAD

**Issue by maks** [xiphonics/picoTracker#181]
Some users of picoTracker as well as LGPT (and M8) use the convention of having empty chains as `00` or `FF` so its helpful if these can be displayed in a different colour. 

This colour should also be user configurable, for now via `config.xml`.

**Comment by democloid** [comment]
Wonder if it would be quick enough to calculate this on the fly and mark on a certain color any empty chains.

**Comment by maks** [comment]
@democloid yes I think we could do that, but if prefer to save cpu cycles on this for other things as it's just a cosmetic and would a small, light touch change just to do it for a few specific chain numbers.

**Comment by RAWJUNGLE** [comment]
> This could also include a separate colour for FE which I think is also commonly used as a kill all chain.

FE is not used as Kill All chain and etc. 

It would be better not to highlight empty chains. 

In those chains where there are no notes in the phrase (but there are commands like kil) do not make white, but highlight gray (for example) as inactive.

**Comment by maks** [comment]
@RAWJUNGLE Thanks for the feedback but would you like to provide more details on your statement of:
>FE is not used as Kill All chain and etc.

as I my understanding is that someone people do use that as the convention.

And I'll just take:
>It would be better not to highlight empty chains.

as your personal preference. Given that the colours for this functionality like all the other colours would be user configurable anyone who didn't want to have these specific chains highlighted could simply choose to use the same colour as used for all other chains.

**Comment by maks** [comment]
At a minimum will support the set of colours [already added](https://github.com/djdiskmachine/LittleGPTracker/blob/02df48e384915c4e735a927fd05e5cd7e639ea46/docs/wiki/config_xml.md?plain=1#L80-L85) to @djdiskmachine  LGPT fork

**Comment by djdiskmachine** [comment]
FE vs 00 is like discussing tab vs double space in programming. Don't go there ;)

**Comment by talktopag** [comment]
I'm a big fan of this idea. Either 00 or FE. A "padding"  pattern is really useful!
